src/components/Checkbox/checkbox.tsx

```diff
@@ -0,0 +1,30 @@
+"use client";
+
+import React, { useState } from "react";
+import Style from "./checkbox.module.css";
+
+interface CheckboxProps {
+  checked: boolean;
+  onChange: () => void;
+}
+
+export default function Checkbox() {
+  const [isChecked, setIsChecked] = useState(false);
```

Comment:
With this, the state will be reset every time you reload the page. What we want is for the state to be persisted based on what we get from the API (item.completed). So you should be using the checked prop instead. For now you can remove this and implement the toggling logic as part of ticket #4